UnitsNet/UnitDefinitions/Permeability.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "Permeability",
+  "BaseUnit": "HenryPerMeter",
+  "XmlDoc": "In electromagnetism, permeability is the measure of the ability of a material to support the formation of a magnetic field within itself.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Permeability_(electromagnetism)",
+  "Units": [
+    {
+      "SingularName": "HenryPerMeter",
+      "PluralName": "HenriesPerMeter",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
```

Comment:
Any prefix units?
Any of these units?
https://www.calculand.com/unit-converter/?gruppe=Permeability&einheit=Henry+per+metre+%5BH%2Fm%5D